docs/mkdocs/docs/api/macros/index.md

```diff
@@ -19,6 +19,7 @@ header. See also the [macro overview page](../../features/macros.md).
 - [**JSON_HAS_FILESYSTEM**<br>**JSON_HAS_EXPERIMENTAL_FILESYSTEM**](json_has_filesystem.md) - control `std::filesystem` support
 - [**JSON_HAS_RANGES**](json_has_ranges.md) - control `std::ranges` support
 - [**JSON_HAS_THREE_WAY_COMPARISON**](json_has_three_way_comparison.md) - control 3-way comparison support
+- [**JSON_DISABLE_ENUM_SERIALIZATION**](json_disable_enum_serialization.md) - switch off default parse/serialization functions for enums
```

Comment:
Please add this line such that the list is sorted by name.

Reply:
added